### PR TITLE
Fee waiver instructions

### PR DESCRIPTION
**Pull request by jazmynlatimer**

### CODE REVIEW

File: frontend/build/css/main.css [hunk 1]
```diff
@@ -4659,6 +4655,9 @@ body.admin .top_bar {
 li.rap_detail {
   margin-bottom: 10px;
 }
+pre.address {
+  overflow: visible;
```
Comment (bengolder) [inline frontend/build/css/main.css]: Not necessary, should either not use the `pre` tag (preserves whitespace) or should abbreviate lines.

File: templates/rap_sheet_instructions.jinja [hunk 2]
```diff
@@ -13,139 +13,260 @@
...
+              <li class="rap_detail">
+                {{ _("EBT benefits card  ") }}
+              </li>
+            </ul>
+        </li>
+        <li>
+          {{ _("Mail your completed Fee Waiver Application form and copies of your government "
+            "benefits (if you are receiving benefits) to this address: ") }}
+            <pre class="address">
+      California Department of Justice
```
Comment (bengolder) [inline templates/rap_sheet_instructions.jinja]: This address is also formatted using `pre` tag. If we change address formatting, should be consistent across the board.

File: templates/rap_sheet_instructions.jinja [hunk 3]
```diff
@@ -13,139 +13,260 @@
...
+      </ol>
+    </div>
+  </div>
+</section>
+
+<section class="rap_ca_instructions">
+  <div class="row">
+    <div class="col-xs-12">
+      <h4>
+        <span class="glyphicon glyphicon-circle-arrow-right" aria-hidden="true"></span>
```
Comment (bengolder) [inline templates/rap_sheet_instructions.jinja:127]: unicode before glyphicons, but not a big deal

File: templates/rap_sheet_instructions.jinja (hunk 2 above)
Comment (jazmynlatimer) [reply templates/rap_sheet_instructions.jinja]: fixed using ul and css

File: templates/rap_sheet_instructions.jinja (hunk 3 above)
Comment (jazmynlatimer) [reply templates/rap_sheet_instructions.jinja:127]: will do later for all glyphicons in CMR

**Comment by bengolder** [comment]
Looking good @jazmynlatimer

**Pull request merged**